papyri/ts.py

```diff
@@ -578,20 +578,30 @@ def visit_inline_target(self, node, prev_end):
         return [Unimplemented("inline_target", self.as_text(node))]
 
     def visit_directive(self, node, prev_end=None):
+        """
+        Main entry point for directives.
+
+        Parses directive arguments, options and content into a MMystDirective
+        object.
+
+        Parameters
+        ----------
+        node: Node
+            The directive to parse
+        prev_end: Unknown
```

Comment:
I couldn't really understand what this is for, maybe we should add a description here.

Reply:
I don't think this is used anymore, this was to work around a tree-sitter parsing issue, where the byte offset position of the previous node what needed to determine the indentation level of a block if memory my memory serves.

Basically the same problem you get with docstring where often the
first line is not indented if it's just after a triple backtick. 

As we have a generic tree visitor, even if a single method needs the parameter, we need to add it to all the methods. 

I think we can mark it for removal to see if it is still used, but I can't find any usage anymore. I'll open an issue once this is merged, to think about removing it.